api/v1/testkube.yaml

```diff
@@ -2791,6 +3009,102 @@ components:
               type: string
             example: ["logline1", "logline2", "logline3"]
 
+    TestTrigger:
+      type: object
+      required:
+        - resource
+        - resourceSelector
+        - event
+        - execution
+        - testSelector
+      properties:
+        name:
+          type: string
+          description: test trigger name
+          example: "test1"
+        namespace:
+          type: string
+          description: test trigger namespace
+          example: "testkube"
+        labels:
+          type: object
+          description: "test trigger labels"
+          additionalProperties:
+            type: string
+          example:
+            env: "prod"
+            app: "backend"
+        resource:
+          type: string
+          description: Kubernetes resource for which we are creating trigger
+          example: pod
+        resourceSelector:
+          $ref: "#/components/schemas/TestTriggerSelector"
+        event:
+          type: string
+          description: listen for event for selected resource
+          example: modified
+        action:
+          type: string
+          description: action to perform when an event matches this trigger
+          example: run
+        execution:
+          type: string
+          description: TestKube resource on which an action will be executed
+          example: testsuite
+        testSelector:
+          $ref: "#/components/schemas/TestTriggerSelector"
+
+    TestTriggerSelector:
+      type: object
+      properties:
+        name:
+          type: string
+          description: kubernetes resource name selector
+          example: nginx
+        namespace:
+          type: string
+          description: resource namespace
+          example: testkube
+        labels:
```

Comment:
Why do we use labels instead of selector? Selector is a standard k8s approach for selecting resources